lib/aws-record/record/version.rb

```diff
@@ -1,18 +1,7 @@
-# Copyright 2015-2017 Amazon.com, Inc. or its affiliates. All Rights Reserved.
-#
-# Licensed under the Apache License, Version 2.0 (the "License"). You may not
-# use this file except in compliance with the License. A copy of the License is
-# located at
-#
-#     http://aws.amazon.com/apache2.0/
-#
-# or in the "license" file accompanying this file. This file is distributed on
-# an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
-# or implied. See the License for the specific language governing permissions
-# and limitations under the License.
+# frozen_string_literal: true
 
 module Aws
   module Record
-    VERSION = '2.7.0'
+    VERSION = File.read(File.expand_path('../../../../VERSION', __FILE__)).strip
```

Comment:
I think we should release a patch version of the gem w/ this fix (so, we should a changelog entry for this)

Reply:
Done!